setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 name = pyhf
 version = 0.5.2
-description = (partial) pure Python HistFactory implementation
+description = pure-Python HistFactory implementation with tensors and autodiff
```

Comment:
@lukasheinrich @kratsg Maybe we should update our PyPI description at the same time? Seems that it isn't too much of a stretch to drop the partial at this point.

Reply:
Going with "pure-Python HistFactory implementation with tensors and autodiff"